### ISSUE TITLE
Writing > Theme enhancements: remove redundant option

### ISSUE THREAD

**Issue by MichaelArestad** [Automattic/jetpack#6218]
Remove the "Scroll infinitely (Shows 7 posts on each load)" option. It seems redundant.

![image](https://cloud.githubusercontent.com/assets/1123119/22389947/909f1660-e4a5-11e6-9782-c3eee59fab56.png)

**Comment by eliorivero** [comment]
It's not redundant. When it's unticked, you don't get an automatic loading of entries as you scroll down. Instead, you get a button at the end of the posts labeled `Older Posts`. After you click it, more posts are loaded.
Another difference is that clicking the button will load only the number of entries defined in Settings > Reading > Blog pages show at most.

I think the checkbox label should be modified since it's very hard to know what it does.

**Comment by eliorivero** [comment]
@richardmuscat @michelleweber can you please have a look at this and suggest a better text for the label. See my explanation about this setting above.

**Comment by MichaelArestad** [comment]
@eliorivero specific issue for this: https://github.com/Automattic/jetpack/issues/6216

**Comment by michelleweber** [comment]
Seems like there should be two equal boxes to check, rather than the current setup? Because right now the language itself is redundant. Seems like the options should be:

- Infinite Scroll: Automatically load new posts as a visitors scrolls (shows 7 posts on each load)
         - Count each new load as a page view.
- Manual Reloading: Require visitors to click a "view more" button to load additional posts.

**Comment by MichaelArestad** [comment]
@michelleweber I'm glad you suggested that. I was thinking something along those lines, though the sub option applies to both. 

Slightly tweaked proposal:

* Automatically load new posts as a visitors scrolls (shows 7 posts on each load)
 * Require visitors to manually load more posts by clicking a button
 * Count each new load as a page view

It turns out the button text can change based on the theme. (I think) If not, it's "older posts."

**Comment by michelleweber** [comment]
Language-wise, that sounds good. Only thought: if the "count each new load as a view" option is under/subordinate, I feel like we should be clear that it applies to both, maybe just with an "applies to both infinite scroll and manual load" parenthetical.

Layout-wise, I don't really understand why the "click a button" option would be sub-option to the infinite scroll option -- they seem like two equal choices?

**Comment by MichaelArestad** [comment]
@michelleweber This is a bit of a conundrum.

I see what you mean about them being equal choices the way they are currently phrase. The funky part of that is, if they are equal choices, what happens when both are selected? Would the user expect full infinite scroll or the button?

What I (poorly) attempted to do was phrase it so the manual page load option was a subsetting or part of the previous option as a sort of limiting option. Maybe something more like this:

* Automatically load new posts as a visitors scrolls (shows 7 posts on each load)
  * Require visitors to clicking a button to show the next "page"
  * Count each new load as a page view

**Comment by MichaelArestad** [comment]
Also, thank you for help with this @michelleweber.

**Comment by michelleweber** [comment]
Hmm. I guess when you picked one, the other would have to gray out/become somehow unavailable, or this needs to be some kind of drop-down/other UI interface where it's clear you can only pick one.

In your last version about, what if i don't want IS, so I uncheck that, and then try to select the "click a button" option? Does it let me? If I can't, that seems confusing to be as a user; if I can, then it's confusing that it's as a setting under a thing that I'm not picking.

Maybe I'm confused about the original UI, so lemme take a step back. 
- Right now, we have a box for "automatically load more content" and  box for "infinite scroll." 
- These both sound like the same thing. If they are, we can delete one. If they're not, we need to change some language to clarify what each one does. But right now, I'm not clear on what that is. So:
- If I check "automatically load more content" but NOT "infinite scroll," what happens to my site?

**Comment by eliorivero** [comment]
@michelleweber 

> If I check "automatically load more content" but NOT "infinite scroll," what happens to my site?

what happens is that when you reach the bottom of a posts listing page, new entries will not be automatically loaded. Instead, there will be a button at the end labeled Older Posts. If you click it, more posts will be loaded.

I'd like to take another step back and consider what this feature does, maybe it will give a better overview of it:
- this feature allows a user to load more entries in the same page
- this can be done automatically, as the user approaches the end of the page, or manually, by clicking a button

Here's a proposal:

- Load new content in the same blog posts page
    - [ ] Display a button to manually load more content
    _Automatically load new posts as user scrolls (shows 7 posts on each load)_
    - [ ] Count each new load as a page view

When `Display a button to manually load more content`, the explanation would be updated to

- Load new content in the same blog posts page
    - [x] Display a button to manually load more content
    _Manually load new posts (shows 10 posts on each load)_
    - [ ] Count each new load as a page view

**Comment by michelleweber** [comment]
Okay, so that means the language we have right now is misleading, because when we say "automatically load more content" what we actually mean is "people will have to click a button." Clarity! Thank you :)

One more question: what is the current default? Ie, on my blog, I have footer widgets, so I automatically have an "older posts" button -- it's not something I had to choose. I ask because I want to be clear whether the options here are nothing/button/IS, or just button/IS.

If the options are nothing/button/IS, then it seems we'd want a heading sentence + choices:

> Allow your readers to load more posts when they reach the bottom of your blog page.
> - Automatically load more posts as the reader scrolls, or
> - Require the reader to click a button to load more posts.
> 
> - When a reader loads more posts, count this as a page view.

The "or" helps clarify, but ideally when you choose one the other become unavailable, to make it super-clear.

If the options are just button/IS, then you only really need a toggle to activate whatever the non-default behavior is, plus the "count as page view" option.

**Comment by eliorivero** [comment]
Michelle, these controls were changed to toggles to match those found in Calypso

<img width="729" alt="captura de pantalla 2017-02-16 a las 16 58 43" src="https://cloud.githubusercontent.com/assets/1041600/23038605/696bba3e-f469-11e6-889c-81e140055b46.png">

The options idea is nice, however for now let's stick to the toggles, since changing these to options will involve a significant update in how the options are handled when they're saved or loaded.

We can keep the heading sentence for the main toggle and the page view counts, and the 
> Automatically load more posts as the reader scrolls, or
> Require the reader to click a button to load more posts.
should be combined in one.

**Comment by rickybanister** [comment]
We can remove the 'track each scroll load' option, and make it on all the time. We can then simply explain in the info bubble that every seven posts count as a page load in GA. No one would want to not track page loads with GA.

**Comment by beaulebens** [comment]
We talked about this one a lot today on our check-in call. I think we came to the conclusion(s) that:

- "Track each scroll..." can be removed and just always be turned on. We can't really imagine anyone who _wouldn't_ want to count these additional-content-loads as pageviews.
- Once that is removed, there are three possible scenarios:
 1. "Automatically load new content..." is turned off (Infinite Scroll disabled), which technically just means your reader will need to click Next/Previous and page through content like a "normal" site
 2. "Automatically load new content..." is turned on, "Scroll infinitely" is turned off. This means new content can be loaded into the page without a refresh, but your reader will need to click a "Load More" button.
 3. Both options are on, in which case new content is loaded automatically when the reader gets towards the bottom of the page.

Based on the above, I think if we remove the "Track each scoll" option, then we are just left with something pretty easily represented using 3 radio buttons:

( ) Manually navigate between pages
( ) Hit a button to load more content at the end of the page
( ) Automatically load new content when scrolling

I'm not sure if those labels are great, but hopefully they're at least descriptive.

**Comment by michelleweber** [comment]
Suggestions, for still more clarity:

( ) Click a button to load more posts on a new page
( ) Click a button to load more posts on the same page
( ) Automatically load more posts on the same page as the reader scrolls down

**Comment by beaulebens** [comment]
Do we have any "rule" on using "click" vs "tap" or something a little more "interaction mode agnostic"?

The first one is usually (depends on the theme) a link vs a button, but I love the clarity of the second set of labels there.

**Comment by MichaelArestad** [comment]
( ) Use default theme behavior to load more posts
( ) Load more posts on the same page on button press
( ) Automatically load more posts as the reader scrolls down

**Comment by michelleweber** [comment]
I don't think we have a rule, but it's a good question -- I'll chat with the other editors.

**Comment by rickybanister** [comment]
Let's go with this (avoiding the click, press, tap, hit verb altogether):

```( ) Load more posts using the default theme behavior
( ) Load more posts with a button
( ) Automatically load more posts as the reader scrolls down```

**Comment by rickybanister** [comment]
cc @tyxla we'll want to reach parity on WordPress.com with this language as well.

**Comment by tyxla** [comment]
It appears this requires some changes to Jetpack itself first. I'll be happy to implement the updated settings in Calypso once they're implemented in Jetpack.

**Comment by eliorivero** [comment]
Closed in #6692
